electron/js/lib/openGraph.js

```diff
@@ -41,21 +42,37 @@ const fetchImageAsBase64 = url => {
 };
 
 const fetchOpenGraphData = url => {
-  const isHtmlContentTypePromise = new Promise(resolve => {
-    const headerRequest = request.head(url);
-    headerRequest.on('response', response => {
-      const isHtmlContentType =
-        response.headers['content-type'] && response.headers['content-type'].match(/.*text\/html/);
-      resolve(isHtmlContentType);
+  const CONTENT_TYPE_LIMIT = 1e7; // 10MB
+  const parsedUrl = urlUtil.parse(url);
+  const normalizedUrl = parsedUrl.protocol ? parsedUrl : urlUtil.parse(`http://${url}`);
```

Comment:
Request should not happen if no protocol are set.

Reply:
That's actually a fix for a bug we have. That's expected behavior